Update kube-ingress-aws-controller for SAToken refresh

demonCoder95 opened the pull request:
Rolling out the update to `kube-ingress-aws-controller` for rolling out the SAToken refresh functionality from this patch: https://github.com/zalando-incubator/kube-ingress-aws-controller/pull/551

More context in this issue: https://github.com/zalando-incubator/kube-ingress-aws-controller/issues/535

Commit:
Update kube-ingress-aws-controller for SAToken refresh

mikkeloscar commented:
:+1:

demonCoder95 commented:
`e2e-tests` keep failing for various reasons:
1. doc web server failed
2. `RouteGroup` and ALB couldn't be created
3. HPA couldn't scale

Let's see if it works 4th time 🙂

mikkeloscar commented:
Depends on: https://github.com/zalando-incubator/kube-ingress-aws-controller/pull/556

mikkeloscar committed:
Update to ingress-controller v0.13.23

https://github.com/zalando-incubator/kube-ingress-aws-controller/releases/tag/v0.13.23

Signed-off-by: Mikkel Oscar Lyderik Larsen <mikkel.larsen@zalando.de>

mikkeloscar commented:
:+1:

gargravarr commented:
:+1:
